CMakeLists.txt

```diff
@@ -11,13 +11,13 @@ option(MQTT_C_TESTS "Build MQTT-C tests?" OFF)
 list (APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
 # MQTT-C library
-add_library(mqttc STATIC 
+add_library(mqttc STATIC
```

Comment:
Seems that I removed some whitespace at the end of the line, do you want me to revert this?